packages/main/src/themes/Input.css

```diff
@@ -156,6 +156,10 @@
 	visibility: hidden;
 }
 
+:host([readonly]) {
+	border: var(--_ui5_input_readonly_border);
```

Comment:
When setting this parameter in the base .css file, ensure it resolves to a valid value across all themes. Currently, it is defined for all Horizon themes, but in Fiori and Belize themes, the parameter resolves to "none", causing the border to be removed from readonly inputs.

Screenshot from sap_fiori_3_hcw:
<img width="464" alt="Screenshot 2025-02-25 at 14 53 27" src="https://github.com/user-attachments/assets/2021536c-076c-46b3-a9fe-d9a9481ec554" />